src/i18n/en.json

```diff
@@ -237,5 +237,7 @@
     "permission.3": "Architecture",
     "permission.5": "Funding",
     "permission.0": "Recovery",
-    "permission.1": "Root"
+    "permission.1": "Root",
+
+    "dashboard.lockedTokenTooltip": "This token is locked. Colony native tokens are locked and non-transferrable by default to avoid unwanted project token transfer outside of the colony."
```

Comment:
I think this might not need to be `dashboard.` as this may be used in other components other than the ones in dashboard

Reply:
I will replace with 'tooltip.lockedToken' that may make more sense for future use cases.